### PR TITLE
feat: Refs #204 Expose L2 (Euclidean) distance to Javascript

### PR DESCRIPTION
**Pull request by GoWind**
Expose the euclidean distance binding to javascript

### CODE REVIEW

File: javascript/fallback.ts [hunk 1]
```diff
@@ -47,6 +47,28 @@ export function sqeuclidean(
...
 
+/**
+ * @brief Computes the L2 Euclidean distance between two vectors.
+ * @param {Float64Array|Float32Array|Int8Array} a - The first vector.
+ * @param {Float64Array|Float32Array|Int8Array} b - The second vector.
+ * @returns {number} The L2 euclidean distance between vectors a and b.
+ */
+export function euclidean(
+  a: Float64Array | Float32Array | Int8Array,
+  b: Float64Array | Float32Array | Int8Array
```
Comment (ashvardanian) [inline javascript/fallback.ts:58]: [The 8-bit unsigned integer arrays are also supported](https://github.com/ashvardanian/SimSIMD/blob/c0807de559829e64ead0afb2563a279c4993d44f/include/simsimd/spatial.h#L59-L61).

File: javascript/lib.c [hunk 2]
```diff
@@ -82,6 +82,9 @@ napi_value api_cos(napi_env env, napi_callback_info info) {
 napi_value api_l2sq(napi_env env, napi_callback_info info) {
     return dense(env, info, simsimd_metric_sqeuclidean_k, simsimd_datatype_unknown_k);
 }
+napi_value api_l2eu(napi_env env, napi_callback_info info) {
```
Comment (ashvardanian) [inline javascript/lib.c:85]: The convention is to call `l2` and `l2sq` or `euclidean` and `sqeuclidean`. The `l2eu` variant makes the codebase less searchable.

**Review by ashvardanian — CHANGES_REQUESTED** [review]
Thank you @GoWind! Looks mostly good, just a couple of little things to polish 🤗

File: javascript/lib.c (hunk 2 above)
Comment (GovindarajanNagarajan-TomTom) [reply javascript/lib.c:85]: ah, so should I just name it `api_l2` then ?

File: javascript/fallback.ts (hunk 1 above)
Comment (GovindarajanNagarajan-TomTom) [reply javascript/fallback.ts:58]: will add `Uint8` support as well. I guess I can also add Uint8 support for the squared euclidean fn as well ?

File: javascript/lib.c (hunk 2 above)
Comment (ashvardanian) [reply javascript/lib.c:85]: Yes, let's do that.

File: javascript/fallback.ts (hunk 1 above)
Comment (ashvardanian) [reply javascript/fallback.ts:58]: If it's missing - sure ;)

File: javascript/lib.c (hunk 2 above)
Comment (GovindarajanNagarajan-TomTom) [reply javascript/lib.c]: I have updated this to name it to just `api_l2`. It should be hopefully consistent now :)

File: javascript/fallback.ts (hunk 1 above)
Comment (GovindarajanNagarajan-TomTom) [reply javascript/fallback.ts]: Unit8Array support was missing. Looks like the C API supports it, so I added it to `fallback.ts` as well !

File: javascript/lib.c (hunk 2 above)
Comment (GoWind) [reply javascript/lib.c]: I have updated this to name it to just api_l2. It should be hopefully consistent now :)

File: javascript/fallback.ts (hunk 1 above)
Comment (GoWind) [reply javascript/fallback.ts]: Uint8Array support was missing in `fallback.ts`. It seems to be supported with the C API, so I added support in the fallback js code as well !

**Review by ashvardanian — APPROVED** [review]

**Pull request merged**